spec/test_cluster.rb

```diff
@@ -0,0 +1,116 @@
+require "docker"
+
+Docker.url = ENV.fetch("DOCKER_HOST")
+
+class TestCluster
+  DOCKER_HOSTNAME = URI(ENV.fetch("DOCKER_HOST")).host
```

Comment:
I don't think there's a valid default, at least not on OS X :-/ This ENV var has to be set for the CLI Docker client to function anyway.